packages/core/directives/marker.ts

```diff
@@ -173,7 +174,14 @@ export class AgmMarker implements OnDestroy, OnChanges, AfterContentInit {
       this._markerManager.updateTitle(this);
     }
     if (changes['label']) {
-      this._markerManager.updateLabel(this);
+      const nextLabel = changes['label'].currentValue;
```

Comment:
I think we dont make a deep equal check here. The developer has to change the reference of the object if he wants a change. We should add that to the doc of the Input.

Reply:
Ok, sure. A bit overhead to escape possible unwanted loop.